app/Models/RequestNCPO.php

```diff
@@ -14,7 +15,8 @@ class RequestNCPO extends Model
     use SoftDeletes;
     use ModelHelpers;
     use HasApproval;
-
+    use HasReferenceNumber;
+    protected $table = 'request_ncpos';
```

Comment:
mo ingon na tana ko nga no need ang table.
hahah naka NCPO man diay imo model name mark.
No Action Required.

pero naa lain option to fix. aside aning $table
i rename ang Model to RequestNcpo // NOT REQUIRED